packages/components/dashboard/dashboard.scss

```diff
@@ -18,6 +18,10 @@
   display: flex;
 }
 
+.pf-c-card {
+  height: 100%;
+}
+
```

Comment:
which component has this class applied to it ?
are we overriding default class of any pf component ?